tee-worker/identity/service/src/main_impl.rs

```diff
@@ -162,6 +162,11 @@ pub(crate) fn main() {
 		enclave_metrics_receiver,
 	)));
 
+	// init in-memory store, it should be done after the o-call bridge is initialized
+	if let Err(e) = enclave.init_in_memory_state() {
```

Comment:
Should it be called `init_in_memory_store` according to the comment? Or is it actually a more generic call for potential similar initialisation in the future?

Reply:
Yes, the idea was to make it generic in case we need something similar in the future